charms/istio-gateway/src/manifest.yaml

```diff
@@ -318,4 +318,4 @@ spec:
       targetPort: 8443
   selector:
     istio: {{ kind }}gateway
-  type: {{ 'ClusterIP' if kind == 'egress' else 'LoadBalancer' }}
+  type: {{ 'ClusterIP' if kind == 'egress' else ingress_service_type }}
```

Comment:
Lets remove the egress part here also.

Reply:
The rule of the good scout says to leave in the better share than found, but I would like to separate removing "kind" from this charm and adding a new configuration option.